src/types/participant.ts

```diff
@@ -0,0 +1,43 @@
+import { Participant, ResponderStatus } from './activity';
+
+export function isActive(status: ResponderStatus) {
+  return [
+    ResponderStatus.Standby,
+    ResponderStatus.Remote,
+    ResponderStatus.SignedIn,
+    ResponderStatus.Available,
+    ResponderStatus.Assigned,
+    ResponderStatus.Demobilized
+  ].includes(status);
+}
+
+export function isCheckedIn(status: ResponderStatus) {
+  return [
+    ResponderStatus.Available,
+    ResponderStatus.Assigned
+  ].includes(status);
+}
+
+export const reduceActive = (count: number, participant: Participant) => {
+  return count + (isActive(participant?.timeline[0].status) ? 1 : 0);
+}
+
+export const reduceStandby = (count: number, participant: Participant) => {
+  return count + (participant?.timeline[0].status === ResponderStatus.Standby ? 1 : 0);
```

Comment:
This part is also repeated many times and could go into a helper function: 
count + (participant?.timeline[0].status === [status] ? 1 : 0)

Reply:
Same, will refactor duplicated definitions of we're going to merge this.